addons/Yush/Yush.lua

```diff
@@ -75,12 +77,24 @@ output = function()
     if settings.Verbose then
         names[current] = names[current] or 'Unnamed ' .. tostring(current):sub(8)
 
+        output_text_table = {}
+        table.insert(output_text_table, names[current])
+        if settings.VerboseKeys then
+            for key, val in pairs(current) do
+                if type(val) ~= 'string' then
+                    val = names[val] or 'Unnamed'
+                end
+                table.insert(output_text_table, key_combos[key] .. ': ' .. val)
+            end
+        end
+        local output_text = table.concat(output_text_table, '\n')
```

Comment:
If you made the variable list, as described above, you can do `output_text_names:concat('\n')` here. That said, I'm a bit confused about the `\n`. Is multiline-output really desirable for this?

Reply:
At least with a text box, I think the multi-line thing looks better, since it makes one command per line. I'm not sure if there's a more preferred way to get a line break.

Reply:
Hmmm, then maybe concat it depending on the output type:
```lua
        if settings.VerboseOutput == 'Text' then
            label:text(output_text_names:concat('\n'))
        elseif settings.VerboseOutput == 'Chat' then
            log('Changing into macro set %s.':format(output_text_names:concat(' > ')))
        elseif settings.VerboseOutput == 'Console' then
            print('Changing into macro set %s.':format(output_text_names:concat(' > ')))
        end
```

Or whatever separator you think works well for the other two.